smartsnippets/migrations/0004_auto_20151002_0350.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smartsnippets', '0003_change_smartsnippetvariable_description'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='smartsnippet',
+            options={'ordering': ['name'], 'verbose_name': 'Component'},
+        ),
+    ]
```

Comment:
What is that this migration is modifying? According to the docs for `AlterModelOptions`:

     Sets new model options that don't directly affect the database schema
    (like verbose_name, permissions, ordering). Python code in migrations
    may still need them.

So, it seems like it's mostly just a noop, right? It should be quick to run then.